cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -3321,6 +3321,9 @@
           "EFA": {
             "Ref": "EFA"
           },
+          "OS": {
```

Comment:
Minor: If we want to be consistent with other parameters that are passed from main stack into substack we should name this `BaseOS`. If we want to try to differentiate this parameter I would pick a more descriptive name like `ComputeOS` so it's easier to find

Reply:
I will make it as BaseOS if it is Ok with you.